python_utils/testRobot.py

```diff
@@ -146,29 +180,40 @@ def createRobotCommand(robot_id, test, tick_counter, period_fraction):
 		log = "speed = %.2f" % cmd.dribbler
 
 	if test == "rotate":
+		cmd.angularControl = 1
```

Comment:
I think we should rename this variable to something like `useAbsoluteAngle` or something. It should be immediately clear what it means if this variable is true or false